File: check_users.js

import { pool } from './backend/src/libs/db.js';
import dotenv from 'dotenv';
import path from 'path';

dotenv.config({ path: path.resolve('backend/.env') });

async function checkUsers() {
  try {
    const [rows] = await pool.query('SELECT user_id, username, warehouse_id FROM Users');
    console.log('User ID | Username | Warehouse ID');
    console.log('--------------------------------');
    rows.forEach(row => {
      console.log(`${row.user_id} | ${row.username} | ${row.warehouse_id}`);
    });
    process.exit(0);
  } catch (error) {
    console.error('Error:', error);
    process.exit(1);
  }
}

checkUsers();

File: check_users.py

import mysql.connector
import os
from dotenv import load_dotenv

load_dotenv()

try:
    mydb = mysql.connector.connect(
      host=os.getenv("MYSQL_HOST"),
      user=os.getenv("MYSQL_USER"),
      password=os.getenv("MYSQL_PASSWORD"),
      database=os.getenv("MYSQL_DATABASE")
    )

    mycursor = mydb.cursor()

    mycursor.execute("SELECT user_id, username, warehouse_id FROM Users")

    myresult = mycursor.fetchall()

    print("User ID | Username | Warehouse ID")
    print("-" * 30)
    for x in myresult:
      print(f"{x[0]} | {x[1]} | {x[2]}")

except Exception as e:
    print(f"Error: {e}")
